sqlglot/dialects/databricks.py

```diff
@@ -0,0 +1,39 @@
+from sqlglot import exp
+from sqlglot.dialects.spark import Spark
+from sqlglot.helper import list_get
+
+
+def generate_date_delta(self, e):
+    func = "DATEADD" if isinstance(e, exp.DateAdd) else "DATEDIFF"
+    return f"{func}({self.format_args(e.text('unit'), e.expression, e.this)})"
+
+
+def parse_date_delta(exp_class):
```

Comment:
dry this up too

Reply:
Moved to Dialect, would be able to replace the lambda in dialects such as Presto, but it's lacking tests so didn't want to go for it in this pr.